### PR TITLE
empty cluster can withdraw all without liquidation

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const NewRemainingDays = ({ cluster, withdrawState, isInputFilled = null }: Prop
   let warningLiquidationState: boolean;
   const clusterRunWay = cluster.newRunWay ?? cluster.runWay;
   let remainingDays: number = clusterRunWay;
-  const typeOfiIsInputFilled =  typeof isInputFilled === 'boolean';
+  const typeOfiIsInputFilled = typeof isInputFilled === 'boolean';
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx:27]: this is redundant
use !!isInputFilled instead for if statement

File: src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx [hunk 2]
```diff
@@ -56,36 +56,36 @@ const NewRemainingDays = ({ cluster, withdrawState, isInputFilled = null }: Prop
...
+            <LiquidationStateError marginTop={'16px'} errorType={errorType}/>
+          </Grid>
+        )}
       </Grid>
-    );
+    </Grid>
+  );
 };
 
 export default observer(NewRemainingDays);
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx:91]: remove observer - nothing to be observed in this component

File: src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx (hunk 1 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx]: +

File: src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/common/NewRemainingDays/NewRemainingDays.tsx]: +

**Pull request closed without merging**